packages/specification/schemas/requests/dispensing/Withdraw.yaml

```diff
@@ -0,0 +1,90 @@
+type: object
+properties:
+  resourceType:
+    type: string
+    enum: [Task]
+  id:
+    type: string
+    example: 6a2624a2-321b-470e-91a6-8ae7a065e2f0
+  contained:
+    type: array
+    items:
+      oneOf:
+      - $ref: ../../components/PractitionerRole/TaskPractitionerRole.yaml
+      - $ref: ../../components/Organization/DispenseNotificationOrganization.yaml
+  extension:
+    type: array
+    description: |
+      FHIR extension array to hold Repeat Information.
+      This identifies the instance of the eRD prescription that is being returned.
+      Given that all eRD prescriptions have the same prescription ID,
+      the combination of Prescription ID and repeat number identifies the issue to withdraw or return.
+    items:
+      $ref: ../../components/_extensions/EPSRepeatInformation.yaml
+  identifier:
+    description: The unique identifier for the Withdraw Request.
+    type: array
+    items:
+      $ref: ../../components/_fragments/identifiers.yaml#/components/schemas/Rfc4122
+  status: 
+    type: string
+    example: in-progress
```

Comment:
this is the only value that's valid for a withdraw not just an example, so should be enum